packages/react/src/ButtonGroup/ButtonGroup.module.css

```
@@ -0,0 +1,8 @@
+.ButtonGroup {
+  display: inline-grid;
+  gap: 8px;
```

Can we switch this out for the design token equivalent?

```suggestion
  gap: var(--base-size-8);
```